packages/react/src/components/Persona/PersonaInitialsColor.ts

```diff
@@ -109,20 +109,25 @@ function personaInitialsColorToHexCode(personaInitialsColor: PersonaInitialsColo
 
 /** @deprecated Use `getPersonaInitialsColor` */
 export function initialsColorPropToColorCode(props: IPersonaProps): string {
-  return getPersonaInitialsColor(props);
+  const { primaryText, text, initialsColor } = props;
+  return getPersonaInitialsColor(text, initialsColor, primaryText);
 }
 
 /**
  * Gets the hex color string (prefixed with #) for the given persona props.
  * This is the logic used internally by the Persona control.
- * @param props - Current persona props
+ * @param text - name text
+ * @param initialsColor - background color
+ * @param primaryText - primary text
  * @returns Hex color string prefixed with #
  */
-export function getPersonaInitialsColor(props: Pick<IPersonaProps, 'primaryText' | 'text' | 'initialsColor'>): string {
-  // eslint-disable-next-line deprecation/deprecation
-  const { primaryText, text } = props;
-  let { initialsColor } = props;
+export function getPersonaInitialsColor(
```

Comment:
I just realize the original typing would work without need to change.. you can revert change for `getPersonaInitialsColor`